src/overlords/mining/extractor.ts

```diff
@@ -105,6 +105,16 @@ export class ExtractorOverlord extends Overlord {
 	}
 
 	private handleDrone(drone: Zerg): void {
+		//fix: if there is no container, then transfer the minerals yourself!
+		if(!this.container && _.sum(drone.carry) == drone.carryCapacity ){
```

Comment:
we probably just shouldn't spawn any extractors unless there is a container

Reply:
good idea, 
what about changing the amount to zero in wishlist if there is no container, something like this:
const amount = this.mineral && this.mineral.mineralAmount > 0 **&& this.container** ? this.mineral.pos.availableNeighbors().length : 0;